src/App.tsx

```diff
@@ -1,61 +1,105 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import { Todos, TodoWithUser, User } from './types';
+
+const todosWithUsers = (todos: Todos[], users: User[]): TodoWithUser[] => {
+  return todos.map(todo => {
+    const match = users.find(user => user.id === todo.userId);
+
+    return {
+      ...todo,
+      name: match?.name || '',
+      username: match?.username || '',
+      email: match?.email || '',
```

Comment:
```suggestion
      user,
```
Just store the user instead.